src/components/ChatBotMessages/index.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+import { useTranslation } from "react-i18next";
+import womanBot from "./images/woman.svg";
+import womanSeeksHelp from "./images/womanSeeksHelp.svg";
+import "./index.scss";
+import { Redirect } from "react-router-dom";
+
+const CHATBOT_THEME = {
+  background: "#FFFEFC",
+  fontFamily: "Roboto",
+  headerBgColor: "#FFBFB5",
+  headerFontColor: "#fff",
+  headerFontSize: "15px",
+  botBubbleColor: "#C8D7C2",
+  botFontColor: "#fff",
+  userBubbleColor: "#FFBFB5",
+  userFontColor: "#fff",
+};
+
+const BotRedirectMessage = ({ path, message }) => {
+  const { t } = useTranslation();
+  return (
+    <div>
+      <p>{t(message)}</p>
+      <Redirect to={path} />
+    </div>
+  );
+};
+
+const ChatBotMessages = () => {
+  const { t } = useTranslation();
+
+  // You can find chatbot documentation here https://lucasbassetti.com.br/react-simple-chatbot/#/
+  const steps = [
+    {
+      id: "1",
+      message: t("chatBotMessages.welcomeMessage"),
+      trigger: "2",
+    },
+    {
+      id: "2",
+      options: [
+        { value: 1, label: t("chatBotMessages.helpLabel"), trigger: "3" },
+        { value: 2, label: t("chatBotMessages.findLabel"), trigger: "4" },
+        { value: 3, label: t("chatBotMessages.blogLabel"), trigger: "5" },
+        {
+          value: 4,
+          label: t("chatBotMessages.opportunityLabel"),
+          trigger: "6",
+        },
+      ],
+    },
+    {
+      id: "3",
+      component: (
+        <div>
+          <p
+            className="immediateHelpLine"
+            dangerouslySetInnerHTML={{
+              __html: t("chatBotMessages.immediateHelpMessage", {
+                link: '<a href="tel:+1123-456-7890">123-456-7890</a>',
```

Comment:
i would consider replacing this with a real number of some public hotline, it will make your demo more legit